app/templates/signatures/ongoing.hbs

```diff
@@ -59,6 +76,29 @@
           @signMarkingActivity={{signMarkingActivity}}
         />
       </td>
+      <td>
+        {{#let (await (this.getSigningHubUrl signFlow)) as |url|}}
+          {{#if url}}
+            <AuLinkExternal
+              @skin="button-naked"
+              @icon="chevron-right"
+              @hideText={{true}}
+              href={{url}}
+            >
+              {{t "open"}}
+            </AuLinkExternal>
+          {{else}}
+            <AuButton
+              @skin="naked"
+              @icon="chevron-right"
+              @hideText={{true}}
+              @disabled={{true}}
+            >
+              {{t "open"}}
```

Comment:
Is this a placeholder?
Avoid buttons that do nothing.
Perhaps a different button icon & tooltip to indicate the link to signinghub is not working (maybe you have no rights or it got removed from signinghub)